src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -1148,6 +1164,32 @@ private int GetPromptIndexForPoint(ReadOnlyCollection<SnapshotSpan> sourceSpans,
                 }
                 return index;
             }
+            
+            private bool IsInActivePrompt(SnapshotPoint point)
+            {
+                var editableBuffer = ReadingStandardInput ? StandardInputBuffer : CurrentLanguageBuffer;
+                if (editableBuffer == null)
+                {
+                    return false;
+                }
+                                                                           
+                var sourceSpans = GetSourceSpans(point.Snapshot);
+                var index = GetSourceSpanIndex(sourceSpans, point);
+                if (index == sourceSpans.Count)
+                {
+                    index--;
```

Comment:
@amcasey Well, `GetSourceSpanIndex` returns the length of the collection if the point is at the end of the last span. Are you suggesting that the prompt will never be in the last span?